tfx/proto/range_config.proto

```diff
@@ -0,0 +1,59 @@
+// Copyright 2020 Google LLC. All Rights Reserved.
```

Comment:
just confirm, this pr doesn't depends on rangeconfig pr a lot other than this proto config, right?

Reply:
yea it only depends on range_config PR for the proto.